pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -180,6 +180,35 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		},
 	}
 
+	if options.DNSConfig != nil {
+		m.ExtraServicePorts = []kapi.ServicePort{
+			{
+				Name: "dns",
+				Port: 53,
```

Comment:
shouldn't we be using the port from DNSConfig.BindAddress? It is typically port 53, but not in some integration test scenarios (like test/integration/dns_test.go)